src/components/Clock.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable consistent-return */
+
+import React from 'react';
+import './Clock.scss';
+
+export class Clock extends React.Component {
+  state = {
+    date: new Date(),
+    name: '$',
+    visibility: false,
+  };
+
+  constructor() {
+    super();
+    this.toggleClockVisibility = this.toggleClockVisibility.bind(this);
+    this.changeClocksName = this.changeClocksName.bind(this);
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({ date: new Date() });
+      // eslint-disable-next-line
+    console.log(this.state.date.toLocaleTimeString());
+    }, 1000);
+  }
+
+  toggleClockVisibility() {
+    this.setState(state => ({
+      visibility: !state.visibility,
+    }));
+  }
+
+  changeClocksName() {
```

Comment:
use arrow functions, so you don't need to bind methods to `this`

Reply:
thanx